pkg/db/db.go

```diff
@@ -0,0 +1,59 @@
+package db
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/medyagh/gopogh/pkg/models"
+)
+
+// Config is database configuration
+type Config struct {
+	Type string
+	Path string
+}
+
+// datab is the database interface we support
+type datab interface {
+	Set(models.DBEnvironmentTest, []models.DBTestCase) error
```

Comment:
this might be okay ! but maybe consider two Sets for each table...(for now we can go like it)